lib/widgets/theme.dart

```diff
@@ -147,7 +147,6 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
     bgSearchInput: const Color(0xffe3e3e3),
     textMessage: const Color(0xff262626),
     channelColorSwatches: ChannelColorSwatches.light,
-    colorMessageHeaderIconInteractive: Colors.black.withValues(alpha: 0.2),
```

Comment:
I know that I mentioned that pure refactors can be marked as "[nfc]", but this wouldn't be an "[nfc]" commit because we changed the color in the process of replacing the variable with `title`.